File: arrays&hashes/434.rb
# Convert a hash into an array of arrays.
# For example, {"chair" => 100, "book" => 14} becomes [["chair", 100], ["book", 14]].

h =  {"chair" => 100, "book" => 14} 
i = 0 
a = []

h.each do | name, price|
  a << [name, price]
end 
p a

File: arrays&hashes/434.js
// # Convert a hash into an array of arrays.
// # For example, {"chair" => 100, "book" => 14} becomes [["chair", 100], ["book", 14]].

var h = { chair: 100, book: 14 };
var a = [];
Object.entries(h).forEach(function ([name, price]) {
  a.push([name, price]);
});
console.log(a);
